purchase_order_line_menu/views/purchase_order_line_views.xml

```diff
@@ -1,20 +1,165 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Moduon Team
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
 <odoo>
+
+    <record id="purchase_order_line_tree" model="ir.ui.view">
+        <field name="name">purchase.order.line.tree</field>
+        <field name="model">purchase.order.line</field>
+            <field name="inherit_id" ref="purchase.purchase_order_line_tree" />
+        <field name="mode">primary</field>
+        <field name="priority">17</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_id']" position="after">
+                    <field name="state" optional="hide" />
+                    <field name="invoice_status" optional="hide" />
```

Comment:
Can you please fix the indentation?

```suggestion
                <field name="state" optional="hide" />
                <field name="invoice_status" optional="hide" />
```

There are several other parts with "improper" indentation in this file.

Reply:
I think this PR should be done by cherry-picking the upstream commits.

Reply:
Fixed.
Thank you @yostashiro